Gemfile

```diff
@@ -10,6 +10,14 @@ source 'http://repo.release.cerner.corp/internal/rubygems/' if internal
 gem 'berkshelf', '~> 3.0', group: :deployment
 gem 'rubocop', '~> 0.18'
 gem 'foodcritic', '~> 4.0'
+gem 'rspec',  '~> 3.1'
+
+# https://github.com/opscode/chef/issues/2547
+if Bundler.current_ruby.on_19?
```

Comment:
It's a related change, as part of adding tests (since the file I was testing pulls in chef). I thought about splitting the file to not have the chef dependency but we'd still run into this sooner or later.

Chef 12 Omnibus (and ChefDK) comes with Ruby 2.1.4 but Chef 11 Omnibus ships with Ruby 1.9.3. [That's why we run our travis build on both|https://github.com/cerner/cerner_splunk/blob/1.7.0/.travis.yml#L7-L8]. 

When you are on ruby 1.9.3, adding `gem 'chef'` or `gem 'chef ~> 12'` as a dependency will fail to resolve with an install error namely with `ohai 8.0.1` because: `ohai requires Ruby version >= 2.0.0`. We could resolve Chef 11 on Ruby 2, but that somewhat defeats part of the reason for using Ruby 2 (which is what 12 comes with)

It could force local users using Ruby 2 to wind up with both gems installed, but ideally they're using gemsets or `bundle exec` to control what versions are resolved where. (Users on 1.9.3 could not upgrade to 12 due to the resolution problem)